bank-system/src/main/kotlin/ru/quipy/aggregates/AccountAggregateState.kt

```diff
@@ -0,0 +1,44 @@
+package ru.quipy.aggregates
+
+import ru.quipy.api.*
+import ru.quipy.core.annotations.StateTransitionFunc
+import ru.quipy.domain.AggregateState
+import java.util.*
+
+// Service's business logic
+class AccountAggregateState : AggregateState<UUID, AccountManagementAggregate> {
+    private lateinit var accountId: UUID
+    private lateinit var userId: UUID
+    var balance: Int = 0
+    var isClosed: Boolean = false
+
+    var createdAt: Long = System.currentTimeMillis()
+    var updatedAt: Long = System.currentTimeMillis()
```

Comment:
I think all of the attributes should be nullable by default (`lateinit` probably)

because first AccountAggregateState is an EmptyState